### PR TITLE
fix deprecations

### PR DESCRIPTION
**Pull request by abhijithch**
Fixes deprecations on `Julia v0.5`.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -309,7 +310,7 @@ try
     open(tmpf, "w") do f
         print_wrapped(f, text, width=30, fix_sentence_endings=true, break_on_hyphens=false)
     end
-    @test readall(tmpf) == """
+    @test @compat readstring(tmpf) == """
```
Comment (tkelman) [inline test/runtests.jl]: does not need `@compat`

File: src/TextWrap.jl [hunk 2]
```diff
@@ -199,7 +199,7 @@ end
     soh = "" # space on hold
 
     # Preliminary conversion to UTF8
-    text = utf8(text)
+    text = String(text)
```
Comment (tkelman) [inline src/TextWrap.jl]: needs `@compat` for 0.3

File: REQUIRE [hunk 3]
```diff
@@ -1,2 +1,2 @@
 julia 0.3
-Compat 0.7.3
+Compat 0.7.9
```
Comment (tkelman) [inline REQUIRE]: this is not the right version. 0.7.15 introduced `String` in Compat, which you need to import. https://github.com/JuliaLang/Compat.jl/commit/4c8c62d275fd11273124fcf4d76f2513eafa0ccc

**Comment by carlobaldassi** [comment]
Thanks, this however introduces warnings in julia 0.4.
To fix it, you should add the line

```julia
import Compat: String
```
below `using Compat`.

**Comment by carlobaldassi** [comment]
Thanks @abhijithch, and thanks also to @tkelman as always for the review.

**Pull request merged**